share/examples/bitcoin.conf

```diff
@@ -156,6 +156,9 @@
 # Maintain coinstats index used by the gettxoutsetinfo RPC (default: 0).
 #coinstatsindex=1
 
+# Maintain the transaction index, which is useful for exploration and development.
```

Comment:
I would be a bit more descriptive here: "Maintain transaction index for all transactions."

Bitcoin Core by default maintains transaction indexes for "relevant" transactions: https://bitcoin.stackexchange.com/a/37979